opentdf-dev.yaml

```diff
@@ -15,11 +15,17 @@ services:
     enabled: true
   authorization:
     enabled: true
-    url: http://localhost:8888
-    client: "tdf-entity-resolution"
-    secret: "secret"
+    ersUrl: http://localhost:8080/entityresolution/resolve
+    clientid: tdf-authorization-svc
+    clientsecret: secret
+    tokenEndpoint: http://localhost:8888/auth/realms/opentdf/protocol/openid-connect/token
```

Comment:
@elizabethhealy Can we resolve this from the issuer that is set in the auth config some how?

Reply:
possibly, but the current issuer doesnt include the realm, also each service only seems to have access to `srp.Config` its own values in the config

Reply:
ok. we can circle back on this in the future.